src/utils.ts

```diff
@@ -295,4 +300,14 @@ export function createMsgResourcePayloadToSign(payload: Partial<MsgCreateResourc
   return MsgCreateResourcePayload.encode(
     MsgCreateResourcePayload.fromPartial(payload)
   ).finish()
+}
+
+export function getCosmosAccount(kid: string): string {
```

Comment:
@anikitinDSR let's change this to publicKeyHex instead of kid.

Reply:
Mostly agreed with Daev here.

Coincidentally, kid + publicKeyHex are identical by value, but that's not always the case with all key id references.